chartfx-math/src/test/java/de/gsi/math/SimpleDataSetEstimatorsTest.java

```diff
@@ -0,0 +1,125 @@
+package de.gsi.math;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.gsi.dataset.DataSet;
+import de.gsi.dataset.spi.DefaultDataSet;
+import de.gsi.dataset.testdata.spi.TriangleFunction;
+
+/**
+ * Test SimpleDataSetEstimators Utility class
+ * 
+ * @author Alexander Krimm
+ */
+class SimpleDataSetEstimatorsTest {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SimpleDataSetEstimatorsTest.class);
+    private static final int N_SAMPLES = 1024;
+
+    static DataSet triangle;
+    static DataSet emptyDataSet;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5084368)